desktop/apps/consignments2/client/actions.js

```diff
@@ -5,15 +5,19 @@ import { push } from 'react-router-redux'
 
 // Action types
 export const CLEAR_ARTIST_SUGGESTIONS = 'CLEAR_ARTIST_SUGGESTIONS'
+export const CLEAR_ERROR = 'CLEAR_ERROR'
 export const CLEAR_LOCATION_SUGGESTIONS = 'CLEAR_LOCATION_SUGGESTIONS'
 export const HIDE_NOT_CONSIGNING_MESSAGE = 'HIDE_NOT_CONSIGNING_MESSAGE'
 export const INCREMENT_STEP = 'INCREMENT_STEP'
 export const SHOW_NOT_CONSIGNING_MESSAGE = 'SHOW_NOT_CONSIGNING_MESSAGE'
 export const UPDATE_ARTIST_AUTOCOMPLETE_VALUE = 'UPDATE_ARTIST_AUTOCOMPLETE_VALUE'
 export const UPDATE_ARTIST_ID = 'UPDATE_ARTIST_ID'
 export const UPDATE_ARTIST_SUGGESTIONS = 'UPDATE_ARTIST_SUGGESTIONS'
+export const UPDATE_ERROR = 'UPDATE_ERROR'
 export const UPDATE_INPUTS = 'UPDATE_INPUTS'
 export const UPDATE_LOCATION_AUTOCOMPLETE_VALUE = 'UPDATE_LOCATION_AUTOCOMPLETE_VALUE'
+export const UPDATE_LOCATION_CITY_VALUE = 'UPDATE_LOCATION_CITY_VALUE'
+export const CLEAR_LOCATION_DATA = 'CLEAR_LOCATION_DATA'
```

Comment:
Some of this logic will likely have to be tuned in QA, but I wanted to capture the case where you successfully select a result from google maps (when this happens, we immediately set the `location_city`, `location_country`, and `location_state` fields), and then hit backspace or otherwise change your input...

If that happens, we should clear our previous location and wait for you to select another one. This could get annoying if someone submitted something like `New York, NY, United Stat` (and deleted the last two letters for some reason), but I'm not sure if we can really be smarter about it.